test/orchard/inspect_test.clj

```diff
@@ -78,10 +76,26 @@
 
 (deftest inspect-var-test
   (testing "rendering a var"
-    (is (= var-result
-           (-> #'*assert*
-               inspect
-               render)))))
+    (is (= (if datafy?
+             '("Class"
```

Comment:
Looks like you can use `(cond-> '["Class" ,,,] datafy? (conj ,,,))` for DRY

Reply:
Fixed.